### PR TITLE
cs_project.py: pass cleanup=true in deleteProject API

### PR DESCRIPTION
**Pull request by rohityadavcloud**
Pass cleanup=true in the delete project module, refer:
https://cloudstack.apache.org/api/apidocs-4.18/apis/deleteProject.html

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
cc @resmo

**Comment by resmo** [comment]
@weizhouapache well, didn't test it yet but in the past, cloudstack just ignored unknown flags? So for earlier versions, this shouldn't break backwards compatibility right?

**Comment by weizhouapache** [comment]
> @weizhouapache well, didn't test it yet but in the past, cloudstack just ignored unknown flags? So for earlier versions, this shouldn't break backwards compatibility right?

@resmo 
this PR also works with the earlier cloudstack versions which do not support the `cleanup` flag. 

There will be a warning message in log (just a warning) like
```
Received unknown parameters for command deleteProject. Unknown parameters : cleanup
```

File: plugins/modules/cs_project.py [hunk 1]
```diff
@@ -222,7 +222,8 @@ def absent_project(self):
             self.result['changed'] = True
 
             args = {
-                'id': project['id']
+                'id': project['id'],
+                'cleanup': 'true'
```
Comment (resmo) [inline plugins/modules/cs_project.py:226]: This should actually be a bool like so ` 'cleanup': True`

**Review by resmo — APPROVED** [review]

**Pull request merged**